pkg/controller/uniteddeployment/uniteddeployment_controller.go

```diff
@@ -188,14 +188,18 @@ func (r *ReconcileUnitedDeployment) Reconcile(request reconcile.Request) (reconc
 	nameToSubset, err := r.getNameToSubset(instance, control, expectedRevision)
 	if err != nil {
 		klog.Errorf("Fail to get Subsets of UnitedDeployment %s/%s: %s", instance.Namespace, instance.Name, err)
-		r.recorder.Event(instance.DeepCopy(), corev1.EventTypeWarning, fmt.Sprintf("Failed%s", eventTypeFindSubsets), err.Error())
+		r.recorder.Event(instance.DeepCopy(), corev1.EventTypeWarning, fmt.Sprintf("Failed %s",
+			eventTypeFindSubsets), err.Error())
 		return reconcile.Result{}, nil
 	}
 
 	nextReplicas, effectiveSpecifiedReplicas, ineffectiveReason := GetAllocatedReplicas(nameToSubset, instance)
 	klog.V(4).Infof("Get UnitedDeployment %s/%s next replicas %v", instance.Namespace, instance.Name, nextReplicas)
 	if !effectiveSpecifiedReplicas {
-		r.recorder.Eventf(instance.DeepCopy(), corev1.EventTypeWarning, fmt.Sprintf("Failed%s", eventTypeSpecifySubbsetReplicas), "Specified subset replicas is ineffective: %s", ineffectiveReason)
+		klog.Errorf("Specified subset replicas is ineffective: %s", ineffectiveReason)
```

Comment:
Better to add the ud namespace/name in the error log

Reply:
done